cmd/release/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"time"
+)
+
+var (
+	targets = map[string][]string{
+		"linux": []string{
+			"amd64",
+			"386",
+		},
+		"darwin": []string{
+			"amd64",
+		},
+	}
+)
+
+const (
+	gitRepoURL = "github.com/MeilleursAgents/terraform-provider-ansiblevault"
+)
+
+func main() {
+	var flagAccessToken string
+	flag.StringVar(&flagAccessToken, "access-token", "", "help message for flagname")
+	flag.Parse()
+
+	date := time.Now().Format("2006-02-01_03:04:05PM")
```

Comment:
Move date initialisation closer to its use (line 40)

Reply:
Done 18d2bb5